iOS/Components/Shared/Extensions/CGColor.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  NSColor.swift
+//  Persephone
+//
+//  Created by Daniel Barber on 2019/2/16.
+//  Copyright © 2019 Dan Barber. All rights reserved.
+//
+
+import CoreGraphics
+
+extension CGColor {
+  static let albumBorderColorLight = CGColor.init(srgbRed: 0, green: 0, blue: 0, alpha: 0.15)
+  static let albumBorderColorDark = CGColor.init(srgbRed: 255, green: 255, blue: 255, alpha: 0.15)
```

Comment:
What do you think about defining these colours in an asset catalog so you can reuse them across platforms without dropping down to Core Graphics?

Reply:
I wasn't even aware that I could do that!